nservicebus/messaging/databus.md

```diff
@@ -68,7 +68,22 @@ snippet:MessageWithLargePayloadUsingConvention
 
 NServiceBus DataBus implementations currently behave differently with regard to cleanup of physical attachments used to transfer data properties. `FileShareDataBus` **does not** remove physical attachments once the message is gone. `AzureDataBus` **does** remove Azure storage blobs used for physical attachments once the message is gone.
 
+### Custom cleanup strategy
 
+NServiceBus cannot tell when a file should be deleted, as removing the file depends on your functional needs. There are a large number of scenarios where removing the file can cause a problem. 
+
+- The message can be (functionally) deferred so that the file will be processed later, or it is passed along to another handler. When the file is removed upon committing the transaction, the deferred or passed on message cannot process the file anymore.
+- The file system isn’t transactional. Transactions spawned by NServiceBus might fail, but the file system cannot participate in them, resulting in a deleted file even when the transaction is aborted.
+- If the outbox feature in NServiceBus is enabled, the message will be removed from the incoming queue, but it might not actually have been processed yet. 
+
+The business requirements should specify when the file should be processed and when it can be removed. Therefor the business should come up with a strategy to remove the files. An option is a strategy like removing the file after (Max(SLA) + x-days), so that it is unlikely that the file still hasn’t been processed.
```

Comment:
typo on Therefore. 

Suggestion:
Evaluating the business requirements can indicate how long the processing of a message should take and when the attachments can safely be removed. One strategy to deal with these attachments is to set up a cleanup policy which removes any files after a certain number of days has passed based on business SLA.